packages/mdc-switch/_mixins.scss

```diff
@@ -23,18 +23,101 @@
 @import "@material/theme/mixins";
 @import "@material/ripple/mixins";
 @import "@material/rtl/mixins";
+@import "@material/elevation/mixins";
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
+@import "./functions";
 @import "./variables";
 
-@mixin mdc-switch-toggled-on-color($color) {
+
+//
+// Public
+//
+@mixin mdc-switch($query: mdc-feature-all()) {
+  // postcss-bem-linter: define switch
+
+  $feat-animation: mdc-feature-create-target($query, animation);
+  $feat-color: mdc-feature-create-target($query, color);
+  $feat-structure: mdc-feature-create-target($query, structure);
+
+  @include mdc-ripple-common($query);
+
+  .mdc-switch {
+    @include mdc-switch-toggled-off-ripple-color($mdc-switch-toggled-off-ripple-color, $query);
```

Comment:
Move this to the end of this block to keep it near the other related color styles (and reduce diff)